docs/roman/pipeline_run.rst

```diff
@@ -70,22 +70,21 @@ Exit Status
 From the Python Prompt
 ------------------------------
 
-You can execute a pipeline or a step from within python by using the
-``call`` method of the class.
-
-The ``call`` method creates a new instance of the class and runs the pipeline or
-step. Optional parameter settings can be specified by via keyword arguments or
+You can execute a pipeline or a step from within python.
+Optional parameter settings can be specified by via keyword arguments or
 supplying a parameter file. Some examples are shown below.
 
 For the exposure pipeline and steps,
 
 ::
 
  from romancal.pipeline import ExposurePipeline
- result = ExposurePipeline.call('r0000101001001001001_01101_0001_WFI01_uncal.asdf')
+ elp = ExposurePipeline()
+ result = elp.call('r0000101001001001001_0001_wfi01_uncal.asdf')
```

Comment:
```suggestion
 elp = ExposurePipeline()
 result = elp.call('r0000101001001001001_0001_wfi01_uncal.asdf')
```
`call` is a class method and doesn't require an instance. Please restore the code from main.